lib/tutorials/ko_KR/getting-started.md

```diff
@@ -18,114 +21,147 @@ _이 튜터리얼은 hapi v11.x.x와 호환됩니다._
-            reply.file('./public/hello.html');
+        handler: (request, h) => {
+
+            return h.file('./public/hello.html');
         }
     });
-});
-
 
+    await server.start();
+    console.log(`Server running at: ${server.info.uri}`);
+};
 ```
 
-위에서 `server.register()` 명령은 inert 플러그인을 Hapi 응용프로그램에 추가합니다. 만약 문제가 발생하고, 그 원인을 알기 위해 `err`를 받고 그 에러를 던지는 익명함수를 전달했습니다. 이 콜백 함수는 플러그인을 등록할 때 필요합니다.
+위에서 `server.register()` 명령은 [inert](https://github.com/hapijs/inert) 플러그인을 hapi 응용프로그램에 추가합니다.
 
-`server.route`() 명령은 서버가 `/hello`에 대한 GET 요청을 받아들이고 `hello.html` 파일의 내용으로 응답하는 `/hello` 경로를 등록합니다. 정적 페이지를 보여주는 inert를 사용하기 _전에_ inert를 등록해야 하므로 inert를 등록하는 라우팅 콜백 함수 안에 두었습니다. 코드가 실행될 때 플러그인이 존재하는 것을 절대적으로 확신할 수 있도록 플러그인을 등록하는 콜백 안에서 플러그인에 의존 코드를 실행하는 것이 일반적입니다.  
+`server.route`() 명령은 서버가 `/hello`에 대한 GET 요청을 받아들이고 `hello.html` 파일의 내용으로 응답하는 `/hello` 경로를 등록합니다. `inert` 플러그인을 등록한 후에 경로 등록을 합니다. 플러그인을 등록한 후 플러그인에 의존하는 코드를 실행하면 코드가 실행될 때 플러그인이 있다는 것을 확신할 수 있습니다.
 
-`npm start`로 서버를 시작하고 브라우저에서 `http://localhost:3000/hello`로 이동하세요. 오 이런! `hello.html` 파일을 만들지 않았기 때문에 404 에러를 받았습니다. 이 에러를 제거하기 위해 빠진 파일을 만들어야 합니다 
+`node server.js`로 서버를 시작하고 브라우저에서 [`http://localhost:3000/hello`](http://localhost:3000/hello)로 이동하세요. 오 이런! `hello.html` 파일을 만들지 않았기 때문에 에러를 받았습니다. 이 에러를 제거하기 위해 빠진 파일을 만들어야 합니다.
 
-`hello.html` 파일을 저장할 `public`이라는 폴더를 디렉터리의 루트에 만들어주세요. `hello.html` 안에 다음 HTML을 넣어주세요.: `<h2>Hello World.</h2>`. 그런 다음 브라우저에서 페이지를 다시불러옵니다. "Hello World."라는 머리말이 보일 것입니다.  
+`hello.html` 파일을 저장할 `public`이라는 폴더를 디렉터리의 루트에 만들어주세요. `hello.html` 안에 다음 HTML을 넣어주세요.: 
 
-Inert는 요청이 있을 때 하드 드라이브에 저장 모든 콘텐츠를 제공합니다. 이는 실시간 다시 읽기 동작을 초래합니다. 취향에 맞춰 `/hello`에 대한 페이지를 바꿔보세요. 
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hapi.js is awesome!</title>
+  </head>
+  <body>
+    <h1>Hello World.</h1>
+  </body>
+</html>
+```
 
-정적 콘텐츠 제공 방법에 대한 자세한 내용은 [Serving Static Content](/tutorials/serving-files)에 자세히 설명되어 있습니다. 이 기술은 웹 응용프로그램에서 이미지, 스타일 시트 그리고 정적 페이지를 제공하는 데 일반적으로 사용됩니다.
+간단하고 유효한 HTML5 문서입니다.
 
-## 플러그인 사용하기
+이제 브라우저에서 페이지를 다시 불러옵니다. "Hello World."라는 머리말이 보일 것입니다.
 
-웹 응용프로그램을 만들때 일반적인 요구 사항은 접근 로그입니다. 응용프로그램에 기본 로깅을 추가하기 위해 [good](https://github.com/hapijs/good) 플러그인과 [good-console](https://github.com/hapijs/good-console)을 서버에 로드해봅니다. 또한 기본 필터링 구조가 필요합니다. 기본 이벤트 타입과 태그 필터링이 필요하기 때문에 [good-squeeze](https://github.com/hapijs/good-squeeze)를 사용합니다.
+[Inert](https://github.com/hapijs/inert)는 요청이 있을 때 하드 드라이브에 저장된 모든 콘텐츠를 제공합니다. 이는 실시간 다시 읽기 동작을 초래합니다. 취향에 맞춰 `/hello`에 대한 페이지를 바꿔보세요.
+
+정적 콘첸츠 제공 방법에 대한 자세한 내용은 [Serving Static Content](/tutorials/serving-files)에 자세히 설명되어 있습니다. 이 기술은 웹 응용프로그램에서 이미지, 스타일 시트 그리고 정적 페이지를 제공하는 데 일반적으로 사용됩니다.
+
+### 플러그인 사용하기
+
+웹 응용프로그램을 만들때 일반적인 요구 사항은 접근 로그입니다. 응용프로그램에 기본 로깅을 추가하기 위해 [hapi pino](https://github.com/pinojs/hapi-pino) 플러그인을 로드합니다.
 
 시작하기 위해 npm에서 모듈들을 설치합니다.:
 
 ```bash
-npm install --save good
-npm install --save good-console
-npm install --save good-squeeze
+npm install hapi-pino
```

Comment:
Hmmm, I'm not entirely sure whether we want to deviate in content here and describe usage of hapi-pino rather than good.  @arb thoughts?